orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy

```diff
@@ -73,7 +73,7 @@ class DependentPipelineExecutionListener implements ExecutionListener {
     if (pipelinePreprocessors) {
       // Resolve templated pipelines if enabled.
       allPipelines = allPipelines.collect { pipeline ->
-       if (pipeline.type == 'templatedPipeline' && pipeline?.schema != "1") {
+       if (pipeline.type == 'templatedPipeline' && pipeline?.scema != null && pipeline?.schema != "1") {
```

Comment:
`s/scema/schema`?

Reply:
`pipeline?.schema` can just be `pipeline.schema`

Reply:
should we have a migrator somewhere that retroactively sets schema == "1"?

Reply:
doh, i must've deleted a key and saved. strange. let me retest it just to make sure i am not crazy :) thanks @ajordens !

Reply:
ok, all checks out... phew :)